components/grid/selection/overview.md

```diff
@@ -10,196 +10,72 @@ position: 0
+* [Selection and Other Grid Features](#selection-and-other-grid-features)
 
-    protected override void OnInitialized()
-    {
-        GridData = new List<Employee>();
-        for (int i = 0; i < 15; i++)
-        {
-            GridData.Add(new Employee()
-            {
-                EmployeeId = i,
-                Name = "Employee " + i.ToString(),
-                Team = "Team " + i % 3
-            });
-        }
-    }
+## Selection Type
 
-    public class Employee
-    {
-        public int EmployeeId { get; set; }
-        public string Name { get; set; }
-        public string Team { get; set; }
-    }
-}
-````
+You can configure the Grid either for row or cell selection:
+* To allow row selection:
+  * Set the [Grid `SelectionMode` parameter](#selection-mode) or
+  * Add a `<GridSelectionSettings>` tag to the `<GridSettings>` tag, and set the `SelectionType` parameter to the `Row` member of the `Telerik.Blazor.GridSelectionType` enum
+* To allow cell selection:
+  * Add a `<GridSelectionSettings>` tag to the `<GridSettings>` tag, and set the `SelectionType` parameter to the `Cell` member of the `Telerik.Blazor.GridSelectionType` enum
 
-### Example - Select rows with checkboxes only
+## Selection Mode
 
-````CSHTML
-@* Require clicks on the checkboxes for row selection*@
+You can configure the selection behavior by setting the Grid `SelectionMode` parameter to a member of the `Telerik.Blazor.GridSelectionMode` enum. The Grid supports the following selection modes:
 
-<TelerikGrid Data=@GridData Navigable="true"
-             SelectionMode="GridSelectionMode.Multiple"
-             Pageable="true"
-             Height="400px">
-    <GridColumns>
-        <GridCheckboxColumn CheckBoxOnlySelection="true" />
-        <GridColumn Field=@nameof(Employee.Name) />
-        <GridColumn Field=@nameof(Employee.Team) Title="Team" />
-    </GridColumns>
-</TelerikGrid>
+* `None` (the default value)—To disable row or cell selection.
+* `Single`—Allows the user to select only one cell or row at a time. If the user attempts to select multiple cells or rows sequentially, only the most recent selection will be retained.
+* `Multiple`—Allows the user to select multiple rows or cells at a time.
 
-@code {
-    public List<Employee> GridData { get; set; }
+## Selection Options
 
-    protected override void OnInitialized()
-    {
-        GridData = new List<Employee>();
-        for (int i = 0; i < 15; i++)
-        {
-            GridData.Add(new Employee()
-            {
-                EmployeeId = i,
-                Name = "Employee " + i.ToString(),
-                Team = "Team " + i % 3
-            });
-        }
-    }
+When you select a row or a cell, they will be highlighted to notify you that they are selected. By default you can click on a row or on a cell to select it. You can also select a row through a checkbox column. To select multiple rows or cells, use the `Ctrl` or `Shift` key to extend the selection.
```

Comment:
* Is " to notify you that they are selected" necessary?
* Link the checkbox column article.